src/migrations/migrations/28.0.2snap1/1_add-maintenance-window-start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Starting from Version 28 Nextcloud recommends to set a default time window for resource heavy background jobs.
+# See https://docs.nextcloud.com/server/28/admin_manual/configuration_server/background_jobs_configuration.html#maintenance-window-start
+
+# Path to config.php file
+config_file="/var/snap/nextcloud/current/nextcloud/config/config.php"
+
+# Searching text (config name)
+search_text="maintenance_window_start"
+
+# New config text
+insert_text="'maintenance_window_start' => 1,"
+
+# Check if config name exists
+if grep -q "$search_text" "$config_file"; then
+    echo "The text '$search_text' is already available in the config file '$config_file'."
+else
+    # Einfügen des Textes in die vorletzte Zeile der Datei
```

Comment:
Could this be in English too? My German is a little bit rust... :wink:

Reply:
Oh, missed that. Done! :)
Will rebase and squash it tomorrow.